README.md

```diff
@@ -34,7 +34,10 @@ and debug the maintained state in redis.
 docker run --rm --name redis-stack-apiserver -p 6379:6379 -p 8001:8001 redis/redis-stack:latest
 ```
 
-Running the apiserver locally.
+Running the apiserver locally requires a proper `.env.local` file according to
+the structure of `.env.fake` but with real data, depending on your desired use
+case. For instance, if you want to run the apiserver alongside the webclient,
+then you need to configure some Auth Provider relevant information.
```

Comment:
Yesterday we noticed there has not been any mention about the necessary env file so far.